website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -374,6 +374,8 @@ A `default_node_pool` block supports the following:
 
 * `custom_ca_trust_enabled` - (Optional) Specifies whether to trust a Custom CA.
 
+* `custom_ca_trust_certificates` - (Optional) A list of up to 10 base64 encoded CAs that will be added to the trust store on nodes with the `custom_ca_trust_enabled` feature enabled.
```

Comment:
Where does this limitation of 10 come from - is this outlined in the docs somewhere?

```suggestion
* `custom_ca_trust_certificates` - (Optional) A list of up to 10 base64 encoded CAs that will be added to the trust store on nodes with the `custom_ca_trust_enabled` feature enabled.
```

Reply:
Limit is provided in description in MS docs:

https://learn.microsoft.com/en-us/azure/templates/microsoft.containerservice/2023-03-02-preview/managedclusters?pivots=deployment-language-bicep#managedclustersecurityprofile